src/services/wantedcontent.js

```diff
@@ -0,0 +1,94 @@
+import models from '../models';
+import HTTPError from '../exceptions/HTTPError';
+
+const create = async ({ content_id, status, my_score }, user_id) => {
+  const wantedList = await models.wanted_contents.create({
+    user_id,
+    content_id,
+    status,
+    my_score,
+  });
+  return wantedList;
+};
+const read = async (limit, user_id) => {
+  const wantedList = await models.wanted_contents.findAll({
+    where: {
+      user_id,
+    },
+    limit,
+  });
+  if (!wantedList) {
+    throw new HTTPError('Wanted content not found or you don\'t have a permission', 404);
+  }
+  return  wantedList ;
+};
+
+const updatecont = async (content_id, { status, my_score }, user_id) => {
```

Comment:
```suggestion
const updatecont = async ({ status, my_score }, content_id, user_id) => {
```

Benzer özelliğe sahip değerler yan yana olması daha kullanışlı olur